scripts/build-catalogue.in

```diff
@@ -108,8 +108,11 @@ with TemporaryDirectory() as tmp:
         fd.write(cmake)
     shutil.copy2(f'@ARB_INSTALL_DATADIR@/BuildModules.cmake', tmp)
     shutil.copy2(f'@ARB_INSTALL_DATADIR@/generate_catalogue', tmp)
-    sp.run('cmake ..', shell=True, check=True, capture_output=not verbose)
-    sp.run('make',     shell=True, check=True, capture_output=not verbose)
+    if verbose:
+        out, err = (sp.PIPE, sp.PIPE)
+    else:
+        out, err = (sp.DEVNULL, sp.DEVNULL)
+    sp.run('make', shell=True, check=True, stdout=out, stderr=err)
```

Comment:
This particular 'feature' of Python gives me the shivers everytime I see it, but it's the way.